pager.c

```diff
@@ -1339,7 +1339,7 @@ static int format_line (struct line_t **lineInfo, int n, unsigned char *buf,
     {
       if (col + 1 > wrap_cols)
 	break;
-      ++col;
+      col += k;
```

Comment:
I'm still unsure whether this should be `col += wcwidth(replacement_char ());` Let me sleep over it some more :)